AK/Time.h

```diff
@@ -175,6 +156,14 @@ constexpr Tuple<T, u8, u8> days_since_epoch_to_date(T days)
     return { year, month, day };
 }
 
+template<typename T>
+constexpr Tuple<T, u8, u8> seconds_since_epoch_to_date(T seconds)
+{
+    constexpr auto seconds_per_day = 60 * 60 * 24;
+    T day_offset_to_epoch = seconds >= 0 ? seconds / seconds_per_day : floor_div(seconds, seconds_per_day);
+    return days_since_epoch_to_date(day_offset_to_epoch);
```

Comment:
It seems like you're never using the other results, why not keeping `seconds_since_epoch_to_year` and returning `return days_since_epoch_to_date(day_offset_to_epoch)[0];` here?
That should make this commit a bit less invasive.